packages/gatsby/src/commands/build-utils.ts

```diff
@@ -70,12 +70,18 @@ export function calcDirtyHtmlFiles(
   const toRegenerate: Array<string> = []
   const toDelete: Array<string> = []
 
+  if (state.html.unsafeBuiltinWasUsedInSSR) {
+    console.warn(
```

Comment:
Can we use reporter here too?
```suggestion
    reporter.warn(
```

Reply:
Will need to add import, but yes